src/SeleniumLibrary/keywords/window.py

```diff
@@ -219,10 +219,10 @@ def set_window_position(self, x, y):
 
     def _log_list(self, items, what='item'):
         msg = [
-            'Altogether {} {}.'.format(
-                len(items), what if len(items) == 1 else '{}s'.format(what))
+            'Altogether %s %s.'
+            % (len(items), what if len(items) == 1 else '{}s'.format(what))
```

Comment:
This would be cleaner with `'Altogether %s %s%s.' % (len(items), what, ...)` where `...` can either be `'' if len(items) == 1 else 's'` or `plural_or_not(items)` if SL has exposed this utility from `robot.utils`.

Reply:
You are correct. SL actually use's `plural_or_not` already in some keyword logging, but I keep forgetting it.